### ISSUE TITLE
Add missing concept to cancer modifier vocabulary - 'Metastasis to lymph node'

### ISSUE THREAD

**Issue by rtmill** [OHDSI/Vocabulary-v5.0#645]
Request to be added:

1) the concept

NAME | CLASS | CONCEPT | VALIDITY | DOMAIN | VOCAB
-- | -- | -- | -- | -- | --
Metastasis to lymph node | Metastasis | Standard | Valid | Measurement | Cancer Modifier

2) fit within hierarchy  - Relationship of 'is a' to '36769180' ('Metastasis')

Looks like there was a SNOMED concept for this (https://athena.ohdsi.org/search-terms/terms/40320620) that was then mapped to 'Secondary malignant neoplasm of lymph node' (https://athena.ohdsi.org/search-terms/terms/318096) but it appears an equivalent concept was not added to the new cancer modifier metastasis hierarchy.  

3) (optional) relationship to SNOMED as parent concept 

Although the relationships between the cancer modifier vocabulary to SNOMED will come in a later submission, this is a good example where this new concept would be a parent concept of the standard SNOMED representation, i.e. 318096 with a 'is a' relationship to this new proposed concept, giving us a full hierarchy. Not sure if it makes sense to do now or wait and tie them all together with one request.

**Comment by vladkorsik** [comment]
Hello! @rtmill , thanks for raising an issue. 
We do not plan to create a concept, because we already have it in the Nodes concept class (OMOP4998920 Distant Lymph Nodes). 
Due to the need to preserve the granularity of the anatomical area level for all cases of lymph node lesions (regional or distant) we would like to create hierarchical groups:

 ![LN_proposal](https://user-images.githubusercontent.com/47443886/176738253-9b762a80-266e-42d5-9d15-394ce699394f.png)

As for SNOMED and Cancer Modifier(CM) interactions, currently we would like to replace the entire set of SNOMED "secondaries" with our "CM Measurements' .
The vocabulary building process, especially for terminologies created by community, is a field for close contact between end users and developers. Please come to our Onco WG sessions to discuss ways to improve ontologies.

**Comment by rtmill** [comment]
Hi again,

@vladkorsik Thanks for the thorough response. The feedback is contradictory to the Oncology WG's understanding of the intention of the 'Metastasis' class of the cancer modifier vocabulary. The reason this came up is that we are mapping all of the relevant NAACCR concepts to the new standard CM Metastasis concepts and this was, at least initially, the only one that was missing. If you look at the overall 'Metastasis' concept, and all of the child concepts, this strategy is already in place. 

In the context of mapping NAACCR to the standard, let's take an example that works. There is [a NAACCR concept](https://athena.ohdsi.org/search-terms/terms/35941752) that has the clear semantic meaning of "metastasis to bone" and we plan to map that to the concept within the new standard representation of the Metastasis hierarchy, i.e. '[Metastasis to Bone](https://athena.ohdsi.org/search-terms/terms/36769301)". 

To put in other words, if the intention of that class is to define the high level classifications of metastases so that one could create a criteria of something like "any kind of metastasis", we would need to fit that into the same hierarchy. That works if we follow the strategy in the example above. If there are exceptions, like in this case where, if I understand you correctly, we are mapping to a concept outside of that hierarchy, and in this case a concept that is just an object "Distant lymph nodes", we lose both the ability to query hierarchically as well as the context of the meaning of source concept. I don't see how the meanings of "Metastasis to lymph nodes" (an action and a site) and "Distant lymph nodes" (purely a site) can be seen as equivalent. 

I'd be happy to bring this up during one of your team meetings or have you join one of the Onc WG meetings to discuss but, unless I've been grossly misunderstanding our conversations with the Odysseus team and the intention of the cancer modifier vocabulary, there is a significant disconnect. 

I'll point some relevant folks to this ticket to weigh in, just to be sure I'm not completely off base here. 

Thanks, Robert

**Comment by tfalcs** [comment]
hi @vladkorsik, thank you so much for attending to Robert's concept request. Would you and @mik-ohdsi be able to attend our next Onc WG next Thursday on our Teams channel? Since this is the first example of a Cancer Modifier concept that we have identified as either missing or inadequate during the mapping process, I think it would be very helpful for all teams involved to come back together to discuss again what we can and should do to ensure the best possible mappings so OMOP databases have the clarity and granularity they need to conduct clear analyses and research. 

Thanks, Thomas

**Comment by vladkorsik** [comment]
@tfalcs , thanks, I will happy to join the meeting. @cgreich we need you here as well.

**Comment by cgreich** [comment]
Friends. 

Metastasis are tumor growths that are distant from the primary site, in contrast to invasive growth to adjacent organs. Affected lymph nodes are inbetween. They are not distant through the blood stream but regional through the lymph. From a prognostic or therapy decisions they are not equivalent to distant metastases. In the TNM staging system they have their own category (N). People sometimes call them “metastasis to lymph node”, but that is still the same thing. 

There are also “distant lymph nodes” (called metastasis or not). They are those not in the organ draining lymph stream. It is not clear how they come about, and phylogenetically they are different from the distant metastases to organs. 

Bottom line: we gave all of them their own concept class “Nodes”. I don’t see a use case to change that. Is there one?

**Comment by rtmill** [comment]
Following up on this. 

@vladkorsik 
I believe the conclusion of our discussion a few weeks back was that the 'node' concepts within cancer modifier would be renamed to include the context of 'spread to' or 'metastasis to' instead of just "distant lymph node". Is that correct? We're finishing up the mappings for this portion of NAACCR and want to be sure we can just assume we can reference the same concept_id and the name will be changed. 

Thanks again for the discussion and help.

**Comment by mgurley** [comment]
@vladkorsik  @rtmill @cgreich @tfalcs 

Hello, just seconding that we would love to get a resolution to this issue.  As we are trying to make changes to the NAACCR ETL to target the Cancer Modifier vocabulary.  And this is a blocker for us.

**Comment by tfalcs** [comment]
This is what I have documented from that meeting:

Decisions:
1.	Regarding lymph node concept ambiguity: Odysseus will make a distinction in the CM vocabulary between regional lymph node spread, which will be OUTSIDE the hierarchy of metastases and distant lymph node spread, which will be INSIDE the hierarchy of metastases. 


I will post this note from our July 7th meeting today to our repo.

**Comment by vladkorsik** [comment]
My apologies for this forum-free window occured.
@mgurley you are right , and the concept [Distant lymph nodes ](https://athena.ohdsi.org/search-terms/terms/36769243) should be used to target Distant nodular spread. The vocabulary will updated  with: a) appropriate renaming and  b) multiaxial hierarchy creation
